YoCode/UnitConverterCheck.cs

```diff
@@ -85,95 +130,66 @@ public List<string> ExtractActions(List<string> actionLines)
             {
-                UnitConverterResults tempActual = new UnitConverterResults();
-                tempActual.input = texts[i];
-
-                for (int j = 0; j < actions.Count; j++)
-                {
-                    tempActual.action = actions[j];
-                    var formContent = GetEncodedContent(i, j);
-
-                    var bar = await client.PostAsync("/Home/Convert", formContent);
-
-                    tempActual.output = await GetResponseAsTaskAsync(bar);
-                    actual.Add(tempActual);
-                }
+                list.Add(Double.Parse(x) * mult);
             }
-            return actual;
-        }
-
-        private FormUrlEncodedContent GetEncodedContent(int i, int j)
-        {
-            return new FormUrlEncodedContent(new[]
-            {
-                        new KeyValuePair<string,string>("text",texts[i]),
-                        new KeyValuePair<string, string>("action",actions[j])
-                    });
-        }
-
-        private static Task<string> GetResponseAsTaskAsync(HttpResponseMessage bar)
-        {
-            return bar.Content.ReadAsStringAsync();
+            return list;
         }
 
-        public void GetResponseAsString(Task<string> bar)
+        public List<string> GetActionKeywords()
         {
-            tempResultStorage = bar.Result;
+            return new List<string> { "action", "value" };
         }
 
-        public List<double> ConvertAnswersToDouble(List<String> strings)
+        public List<double> CheckActions(string action)
         {
-            return strings.Select(s => double.Parse(s)).ToList();
+                foreach (var keywords in KeywordMap)
+                {
+                    if (action.ToLower().ContainsAny(keywords.Key))
+                    {
+                        return keywords.Value;
+                    }
+                }
+            return new List<double>{0.1};
         }
 
-        public List<string> GetActionKeywords()
+        public (double,double) StringToDouble(string a, string b)
         {
-            return new List<string> { "action", "value" };
+            return (Double.Parse(a), Double.Parse(b));
         }
 
         public bool OutputsAreEqual()
         {
             bool ret = true;
-
-            UnitConverterCheckEvidence.GiveEvidence(String.Format("{0,-9} {1,10}","Expected","Actual"));
-
-            for(int i = 0; i < actual.Count; i++)
+            try
             {
-                (var A, var B) = ToDouble(expectedOutputs[i], actual[i].output);
-
-                var x = String.Format("{0,-9} and {1,-9} Are equal: {2,-4} ", Math.Round(A,6), Math.Round(B,6), A.ApproximatelyEquals(B));
-                UnitConverterCheckEvidence.GiveEvidence(x);
+                UnitConverterCheckEvidence.GiveEvidence(String.Format("{0,-9} {1,10}", "Expected", "Actual"));
 
-                if (!A.ApproximatelyEquals(B))
+                for (int i = 0; i < actual.Count; i++)
```

Comment:
There are lots of places in this function where you could use `var` instead of specifying the type name. See if you can find them all. ;)